app/assets/stylesheets/navigation.scss

```diff
@@ -44,6 +44,10 @@
   span {
```

Comment:
does it make sense moving navigation styles to webpack?

Reply:
No, we aim to replace the navigation component later.
Anyway, not in this PR please.